src/test/java/jp/ikedam/jenkins/plugins/updatesitesmanager/DescribedUpdateSiteJenkinsTest.java

```diff
@@ -172,7 +172,6 @@ public void testDoConfigure() throws Exception
         // duplicate id
         {
             WebClient wc = new WebClient();
-            wc.setPrintContentOnFailingStatusCode(false);

```

Comment:
I'm not sure why you remeved this.
I put this line to suppress unuseful outputs (they should be saved to xxxx-output.txt).
(I think I have to see how that behavior changes)